plugin/src/py/android_screenshot_tests/device_name_calculator.py

```diff
@@ -20,8 +20,9 @@
 
 class DeviceNameCalculator:
 
-    def __init__(self, executor=AdbExecutor()):
+    def __init__(self, executor=AdbExecutor(), args = []):
```

Comment:
Instead of arbitrary args, I think it would be better if we explicitly had a property for the serial no